trunk/src/protocol/srs_protocol_raw_avc.cpp

```diff
@@ -400,7 +400,16 @@ srs_error_t SrsRawHEVCStream::mux_sequence_header(std::string vps, std::string s
     //      sequenceParameterSetNALUnit
 
     // use simple mode: nalu size + nalu data
-    int nb_packet = 23 + 5 + (int)vps.length() + 5 + (int)sps.length() + 5 + (int)pps.length();
+    /**/
+	int nPPSSize = 0;
+	for (auto stl_VectorIterator = pps->begin(); stl_VectorIterator != pps->end(); stl_VectorIterator++)
```

Comment:
We do not recommend using the "auto" syntax, as it is a new feature introduced in C++11. Currently, SRS is still using the C++98 standard.